packages/machine-extractor/src/MachineExtractResult.ts

```diff
@@ -708,6 +709,46 @@ export class MachineExtractResult {
     return delays.toObject();
   };
 
+  getAllMachineOptions = () => {
```

Comment:
we are not really super strict about it today but I think that currently we call those `implementations` (and not `options`)